.travis.yml

```diff
@@ -1,6 +1,7 @@
+sudo: false
```

Comment:
does this make the VM start up faster?

Reply:
No, but waiting times are less in travis.